sdk/eventgrid/azure-eventgrid/azure/eventgrid/_publisher_client.py

```diff
@@ -113,27 +117,24 @@ def send_events(self, events, **kwargs):
         if not isinstance(events, list):
             events = cast(ListEventType, [events])
```

Comment:
do we want to replace this cast/wrapper of non-list events with raising `ValueError` as in the description we say that we only accept list of events?

Reply:
we still want to accept a single event - the purpose of removing from documentation is that we want to encourage users to use a list instead - there's no reason not to accept a single event otherwise